Core/Core/UIViews/CoreWebView.swift

```diff
@@ -710,3 +739,103 @@ extension CoreWebView {
             .replacingOccurrences(of: ">", with: "&gt;")
     }
 }
+
+extension CoreWebView {
```

Comment:
Since this file is already pretty large maybe we could extract it into an extension file.